codegen/scala-gen/src/test/scala/com/akkaserverless/codegen/scalasdk/ValueEntitySourceGeneratorSuite.scala

```diff
@@ -140,7 +140,7 @@ class MyValueEntityProvider private(entityFactory: ValueEntityContext => MyValue
     new MyValueEntityHandler(entityFactory(context))
 
   override final val additionalDescriptors =
-    service.MyServiceProto.javaDescriptor :: Nil
+    DomainProto.javaDescriptor :: service.MyServiceProto.javaDescriptor :: com.external.ExternalDomainProto.javaDescriptor :: Nil
```

Comment:
yeah, I had a hunch that something was missing there

Reply:
aha!